src/WordPress/PluginRoutes.php

```diff
@@ -23,6 +23,16 @@ class PluginRoutes
             ),
         ),
 
+        // Must be above "plugin/:id/settings/:human_readable_id"
+        'plugin/:id/settings/default_settings' => array(
```

Comment:
Instead of hard coding this route just have `plugin/:id/settings/:human_readable_id` go to a method that switches on `:human_readable_id` and then returns the method that handles that ID.  This functionality working correctly as is depends on the order of these routes being correct which has a lot of potential for bugs.